doc/nrf/protocols/bt/bt_solutions.rst

```diff
@@ -0,0 +1,82 @@
+Bluetooth solution areas
+########################
+
+Bluetooth technology provides full stack support for a wide and expanding range of use cases.
+
+Audio streaming
+***************
+
+|NCS| supports the Bluetooth LE Audio standard, including Auracast broadcast audio.
+See :ref:`nrf53_audio_app` to get started.
+
+Note that Nordic Semiconductor SoCs do not support Bluetooth Classic Audio, which is based on the
+Bluetooth BR/EDR radio.
+
+Data transfer
+*************
+
+Data transfer is the main use case area for Bluetooth LE and covers use cases like wearables, PC peripherals, and
```

Comment:
We should move "Data transfer" to be the first listed use case as this is the main use case

Reply:
The order follows the what you see on the Bluetooth SIG website. No reason to follow it strictly, but Data transfer is well visible on the screen anyway.